app/models/user.rb

```diff
@@ -38,6 +38,7 @@ def deleted
 
   validates :email, :encrypted_password, :authentication_token, absence: true, if: :built_in?
   schema_validations except: [:encrypted_password]
+  validates :time_zone, inclusion: { in: ActiveSupport::TimeZone.zones_map.keys }, allow_nil: true
```

Comment:
Yes the exception will be raised, only that user will be affected. Yah I didn't handle that case, thanks I will add an issue regarding this.